be/server/src/main/java/com/baseball/dto/BatterDto.java

```diff
@@ -0,0 +1,58 @@
+package com.baseball.dto;
+
+public class BatterDto {
+
+    private String name;
+    private Integer plateAppearances;
+    private Integer hit;
+    private Integer out;
+    private Float average;
```

Comment:
```suggestion
    private final String name;
    private final Integer plateAppearances;
    private final Integer hit;
    private final Integer out;
    private final Float average;
```

dto 안의 멤버는 전부 private final 이 될 수 있도록 해주세요.
dto 는 불변객체이어야 합니다.

Reply:
넵넵 변경하겠습니다 :)